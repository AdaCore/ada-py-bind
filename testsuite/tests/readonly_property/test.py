import gen

p = gen.Point()

print p.x
try:
    p.x = 12
except Exception, e:
    print e
