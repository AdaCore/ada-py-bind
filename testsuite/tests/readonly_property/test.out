12
Read only property
